Formula/alpine.rb

```diff
@@ -31,6 +31,9 @@ class Alpine < Formula
     depends_on "linux-pam"
   end
 
+  # patch for macOS obtained from developer; see git commit 701aebc00aff0585ce6c96653714e4ba94834c9c
+  patch :DATA
```

Comment:
```suggestion
  # patch for macOS obtained from developer
  # https://repo.or.cz/alpine.git/commitdiff/701aebc00aff0585ce6c96653714e4ba94834c9c
  patch do
    url "https://repo.or.cz/alpine.git/patch/701aebc00aff0585ce6c96653714e4ba94834c9c"
    sha256 "fe0a41c025c6cd3e8eb3c51c9d0d96d2986eedd0d54ba9fbe29857a77853c0f4"
  end
```
Does this apply?

Reply:
No, I'm afraid it fails; I guess there are other changes to the `configure` file in the history since the release of 2.26.

Reply:
Hm, ok. Let's add a link directly to the upstream commit anyway to make it easier to find.